featurebyte/session/databricks_unity.py

```diff
@@ -91,3 +94,25 @@ def _upload_file_to_storage(self, path: str, src: BinaryIO) -> None:
 
     def _delete_file_from_storage(self, path: str) -> None:
         self._files_client.delete(file_path=path)
+
+    async def list_schemas(self, database_name: str | None = None) -> list[str]:
```

Comment:
Do we have to change the implementation for spark as well?

Reply:
Spark and legacy Databricks (without Unity) used Hive as a metastore, which do not expose information schema tables (metadata is maintained in separate database). There may not be any limits to the SHOW statement in Hive, we will need to test it out.